web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BeregningJson.kt

```diff
@@ -20,15 +21,26 @@ internal data class BeregningJson(
     val fradrag: List<FradragJson> = emptyList()
 )
 
-internal fun Beregning.toJson() = BeregningJson(
-    id = getId().toString(),
-    opprettet = getOpprettet().toString(),
-    fraOgMed = getPeriode().getFraOgMed().format(DateTimeFormatter.ISO_DATE),
-    tilOgMed = getPeriode().getTilOgMed().format(DateTimeFormatter.ISO_DATE),
-    sats = getSats().name,
-    månedsberegninger = getMånedsberegninger().map { it.toJson() }, // TODO show fradrag/month
-    fradrag = getFradrag().map { it.toJson() }
-)
+internal fun Beregning.toJson(): BeregningJson {
+    val epsInputFradragMap = getFradrag()
```

Comment:
Her mangler det vel en periodisering et sted? Dette vil vel ikke fungere hvis perioden for fradraget var f.eks 01.01 - 31.12? (noe input fradragene ofte er)?